pkg/handler/eval.go

```diff
@@ -133,8 +135,14 @@ var LookupFlag = func(evalContext models.EvalContext) *entity.Flag {
 
 var EvalFlagsByTags = func(evalContext models.EvalContext) []*models.EvalResult {
 	cache := GetEvalCache()
+	var operator string
+	if evalContext.FlagTagsOperator == nil {
+		operator = ""
+	} else {
+		operator = *evalContext.FlagTagsOperator
+	}
 
-	fs := cache.GetByTags(evalContext.FlagTags)
+	fs := cache.GetByTags(evalContext.FlagTags, operator)
```

Comment:
for readability, I think you can just pass the `evalContext.FlagTagsOperator` as `*string` to `cache.GetByTags`, and then check if `if operator == nil || *operator == models.EvaluationBatchRequestFlagTagsOperatorANY` there.

Reply:
Thanks reads better